Modules/Scripted/DICOMLib/DICOMUtils.py

```diff
@@ -16,10 +16,11 @@
 ######################################################### 
 
 #------------------------------------------------------------------------------
-# Get patient UID by patient name for easy loading of a patient
 def getPatientUIDByPatientName(name):
+  """ Get patient UID by patient name for easy loading of a patient
+  """
```

Comment:
`if not hasattr(slicer,'dicomDatabase'):` -> `if not hasattr(slicer, 'dicomDatabase'):`

Argument should have a space after ",".

(I will setup python style check shortly)